doc/cmdline-commands-linked.asciidoc

```diff
@@ -0,0 +1,121 @@
+'''
```

Comment:
Please remove this from the repository. It's an intermediate file, no need to commit it.